string name
float64 presence
float64 intensity